src/errors.ts

```diff
@@ -38,6 +38,18 @@ export class GitDirNotInitializedError extends Error {
   }
 }
 
+export class GitNotSetupError extends Error {
+  constructor() {
+    super(
+      `Git has not been setup:\n` +
+        `Run\n` +
+        `1. Run 'git config --global user.name <your name>'\n` +
+        `2. Run 'git config --global user.email <your email>'`
+    )
```

Comment:
Hi, @ivanramosnet I do not know if we can pass the git error from SimpeGit to this error. Is that possible? Ideally we should show the Git error. The original error is:

```
Author identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address (got 'xxx@xxx.(none)')
```

If that's not possible I would use a message more similar to the original. Maybe:

```
Git author identity unknown.
Run:
  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"
to set your account's default identity.
Omit --global to set the identity only in this repository.
```

I don't remember what error the workflow log shows when it fails.

Reply:
OK, I thought it was hidden.